src/container/builder_impl.rs

```diff
@@ -39,7 +39,9 @@ pub(super) struct ContainerBuilderImpl {
 impl ContainerBuilderImpl {
     pub(super) fn create(&mut self) -> Result<()> {
         if let Process::Parent(_) = self.run_container()? {
-            std::process::exit(0);
+            if self.init {
```

Comment:
Can I ask you to add a comment to the struct explaining init?

Reply:
@utam0k Added some more explanations. Let me know what you think.